app/graphql/types/organization_type.rb

```diff
@@ -2,28 +2,34 @@
 
 module Types
   class OrganizationType < Types::BaseObject
+    description 'Organization Type'
+
     field :id, ID, null: false
-    field :name, String, null: false
+
+    field :address_line1, String
```

Comment:
What about grouping these fields in "logical" block (at least for the address block)

```ruby
    field :id, ID, null: false

    field :legal_name, String
    field :legal_number, String
    field :logo_url, String
    field :name, String, null: false

    field :email, String

    field :api_key, String, null: false
    field :webhook_url, String
    
    field :address_line1, String
    field :address_line2, String
    field :city, String
    field :country, Types::CountryCodeEnum, null: true
    field :state, String
    field :zipcode, String

    field :timezone, Types::TimezoneEnum, null: true
    
    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
    field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
```